consensus/polybft/consensus_runtime.go

```diff
@@ -224,7 +224,9 @@ func (c *consensusRuntime) OnBlockInserted(block *types.Block) {
 	}
 }
 
-func (c *consensusRuntime) createCommitmentAndBundles(txs []*types.Transaction) error {
+// getCommitmentFromTransaction gets the registered commitment (if any)
+// from a transaction, and builds bundles for it
+func (c *consensusRuntime) getCommitmentFromTransaction(txs []*types.Transaction) error {
```

Comment:
Rename this to `getCommitmentFromTransactions`?

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1013/commits/1ff7061ac7b16bc09d087c87578b6a4292a157a1